frontend/src/metabase-types/api/entity-id.ts

```diff
@@ -0,0 +1,24 @@
+const NANOID_ALPHABET =
+  "0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
```

Comment:
```suggestion
  "_-0123456789abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
```

This is taken from our Clojure implementation.

https://github.com/metabase/metabase/blob/44888103944f545c3b6c84c489832fdc2868e534/src/metabase/util/jvm.clj#L28